### DISCUSSION TITLE
Socket exception issue

### DISCUSSION THREAD

**Discussion by prabhathulasi** [serverpod/serverpod#973] [Q&A] [answered]
**Describe the bug**
Even the server pod is running in the background flutter showing socket exception error 

**To Reproduce**
1. Go to pod server side and run the server
2. run the flutter app
3. call the api for the server folder
4. SocketException: Network is unreachable (OS Error: Network is unreachable, errno = 101), address = localhost, port = 42354

**Comment by Isakdl** [comment, marked as answer]
Where are you running the flutter app and what platform?

If you are running on a physical device or an Android emulator you cannot use localhost as the address. You can find your IP address with `ifconfig` on linux/macos or `ipconfig` on windows. And a physical device has to be connected to the same local network as your computer (I.E. same wifi). 

If you are running on an Android emulator you can use the address, `10.0.2.2`.
